tutorials/tutorial1.ipynb

```diff
@@ -164,862 +175,1989 @@
-    "\n",
-    "For more information type `python $atacworks/scripts/bw2h5.py --help`\n"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Step 6: Read the validation data and labels and save in .h5 format\n",
-    "\n",
-    "Next we read and save the validation data for the model.\n"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 10,
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "INFO:2020-06-09 20:10:56,268:AtacWorks-bw2h5] Reading intervals\n",
-      "INFO:2020-06-09 20:10:56,410:AtacWorks-bw2h5] Read 1260 intervals\n",
-      "INFO:2020-06-09 20:10:56,410:AtacWorks-bw2h5] Writing data in 2 batches.\n",
-      "INFO:2020-06-09 20:10:56,410:AtacWorks-bw2h5] Extracting data for each batch and writing to h5 file\n",
-      "INFO:2020-06-09 20:10:56,411:AtacWorks-bw2h5] batch 0 of 2\n",
-      "INFO:2020-06-09 20:11:39,943:AtacWorks-bw2h5] Done! Saved to ./Mono.50.2400.val.h5\n"
-     ]
-    }
-   ],
-   "source": [
-    "!python $atacworks/scripts/bw2h5.py \\\n",
-    "           --noisybw dsc.1.Mono.50.cutsites.smoothed.200.bw \\\n",
-    "           --cleanbw dsc.Mono.2400.cutsites.smoothed.200.bw \\\n",
-    "           --cleanpeakbw dsc.Mono.2400.cutsites.smoothed.200.3.narrowPeak.bw \\\n",
-    "           --intervals val_intervals.bed \\\n",
-    "           --out_dir ./ \\\n",
-    "           --prefix Mono.50.2400.val \\\n",
-    "           --pad 5000"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "This produces a .h5 file in the current directory (`Mono.50.2400.val.h5`) containing the validation data for the model.\n",
+    "We provide a chromosome sizes file 'hg19.auto.sizes' that contains sizes for all the autosomes of the hg19 reference genome. We split off chromosome 2 to use as the validation set, and chromosome 10 to use as the holdout set, and use the remaining autosomes as the training set. Since a whole chromosome is too long to feed into the model at once, we split each of these chromosomes into 50,000-bp long intervals.\n",
     "\n",
+    "This command trains a deep learning model using the supplied clean and noisy ATAC-seq data, for 10 epochs (10 full passes through the dataset). At the end of every epoch, the current state of the model is saved in the directory `atacworks_train_latest`, and the performance of the current model is measured on the validation set. At the end, out of the 10 saved models, the one with the best performance on the validation set is saved as `atacworks_train_latest/model_best.pth.tar`\n",
     "\n",
-    "## Step 7: Train and validate a model using the parameters in the given config files\n",
+    "This model has learned a mapping from the 50-cell signal to the 2400-cell signal and peak calls. Given a new 50-cell ATAC-seq track, it can denoise the track and produce high-quality peak calls.\n",
```

Comment:
Move lines 221-225 into a cell below the training command.

Reply:
done.